tests/test_functions.py

```diff
@@ -0,0 +1,136 @@
+#################
+## contains tests for utils in pyspextool
+###############
+
+import numpy as np
+from pyspextool.calibration import *
+from pyspextool.fit import *
+from pyspextool.io import *
+from pyspextool.plotting import *
+from pyspextool.uspex import *
+from pyspextool.utils import *
+
+import pytest
+xfail = pytest.mark.xfail #taking care of tests that are expected to fial
+
+@xfail(reason='examples not implemented')
+def test_avehdrs():
+	pass
+
+@xfail(reason='examples not implemented')
+def test_bicucof():
+	pass
+
+@xfail(reason='examples not implemented')
+def test_bicuval():
+	pass
+
+@xfail(reason='examples not implemented')
+def test_findorders():
+	pass
```

Comment:
change `pass` to `assert = 0`

Reply:
I have changed from pass to assert= 0 and I have changed from xfail to skip